app/views/dashboards/_current_tasks.html.erb

```diff
@@ -29,7 +29,10 @@
             >
               <% MyModuleStatusFlow.find_each do |status_flow| %>
                 <% status_flow.my_module_statuses.each do |status| %>
-                  <option value="<%= status.id %>" <%= 'data-selected=true' unless status.final_status? %> ><%= status.name %></option>
+                  <option value="<%= status.id %>"
+                          data-completion-consequence="<%= status.my_module_status_consequences.where(type: "MyModuleStatusConsequences::Completion").any? %>">
```

Comment:
Do you test with addons? I think it will mark you only one status

Reply:
I tested it. It marks first two statuses.